lib/tty/pager/system.rb

```diff
@@ -123,22 +124,57 @@ def initialize(**options)
       #
       # @api public
       def page(text, &_callback)
-        return text unless output.tty?
+        write(text)
+        wait
+      end
+
+      # Spawn the pager process
+      #
+      # @return [PagerIO]
+      #   A wrapper for the external pager
+      #
+      # @api private
+      def start
```

Comment:
I like your comment and since this is a private API we could provide more intention revealing name like `spawn_pager`?

Reply:
Agreed, that's a much better name.